utils/google.py

```diff
@@ -1,11 +1,10 @@
 from requests_html import HTMLResponse, HTMLSession
 
-from utils.custom_logger import CustomLogger
+from utils.custom_logger import main_logger
 
 BLOCK_STR = "Our systems have detected unusual traffic from your computer network. This page checks to see if it's really you sending the requests, and not a robot. Why did this happen?"
 
-main_logger = CustomLogger().main_logger
-
+logger = main_logger.getChild('utils.google')
```

Comment:
Why do we need to use `getChild()`?

Reply:
Are there any benefits?

Reply:
This way each module has its own logger. Two benefits:
- we can display the name of the logger in the log message if we want to by changing the format in custom_logger.py, which would help with identifying which area of the code a log message came from (and less verbose than the overly long file path, perhaps we could remove that in the future, or e.g. for files specifically)
- you can configure each logger instance different, for example, you may configure one of them to only let warnings and errors be logged, while another also debug and info messages. e.g. if you want to only debug a specific part of the program, changing that module's logger to be more verbose could help